src/CMakeLists.txt

```diff
@@ -396,14 +464,16 @@ add_dependencies(${EXECUTABLE_HELLOREGION} ${COMMON_LIBS})
 #
 # Setup prototest example
 #
-set(EXECUTABLE_PROTOTEST prototest)
-add_executable(${EXECUTABLE_PROTOTEST} examples/prototest.cpp)
-target_link_libraries(${EXECUTABLE_PROTOTEST}
-                      ${COMMON_LIBS})
-set_target_properties(${EXECUTABLE_PROTOTEST} PROPERTIES COMPILE_FLAGS ${COMPILE_FLAGS})
-set_target_properties(${EXECUTABLE_PROTOTEST}
-                      PROPERTIES LINK_FLAGS "${COMMON_LINK_FLAGS}")
-add_dependencies(${EXECUTABLE_PROTOTEST} ${COMMON_LIBS})
+if(SERIALIZATION)
```

Comment:
It looks like you figured this out in a different PR, plus I see on like ~93 that you are using CAPNP_LITE for Windows so this shouldn't be necessary. Please remove the `SERIALIZATION` flag.